third_party/amd/backend/compiler.py

```diff
@@ -396,11 +395,11 @@ def make_amdgcn(src, metadata, options):
         metadata["name"] = names[0]
         # llvm -> hsaco
         flags = []
-        # XXX(Kyle): The sink-insts-to-avoid-spills helps mitigate register spills
-        # under heavy workload, but it can also lead to regression in some cases.
-        # But from existing observation, the regression is not significant.
-        # It would be better to have some heuristics for it.
-        if options.instruction_sched_variant == 'attention':
+        # The sink-insts-to-avoid-spills flag asks LLVM backend to sink instructions
+        # into cycles to avoid register spills in the MachineSinking pass, while it
```

Comment:
".. into loops .." ?

Reply:
It's actually cited from the LLVM MachineSinking pass, but 'loops' is also ok.